resources/monitoring/templates/grafana/dashboards-1.14/etcd.yaml

```diff
@@ -1109,8 +1108,9 @@ data:
                 "30d"
             ]
         },
-        "timezone": "",
-        "title": "Kubernetes / etcd",
+        "timezone": "browser",
+        "title": "etcd",
```

Comment:
We adapted the titles this way. Please use it as is for all the dashboards.

Reply:
Unset the timezones (set to the empty string) in all of the dashboards